nav2_behaviors/include/nav2_behaviors/timed_behavior.hpp

```diff
@@ -45,6 +45,12 @@ enum class Status : int8_t
   RUNNING = 3,
 };
 
+struct ResultStatus
+{
+  Status status;
+  unsigned short error_code{0};  //NOLINT
```

Comment:
Why does this need no linting?

Reply:
^ not much of a reason for this not to just be in unsigned int, really. But I don't really care

Reply:
Switched this to a uint16_t :)